docs/src/pages/index.mdx

```diff
@@ -110,9 +110,9 @@ const DATA = [
 R.pipe(
   DATA,
   R.filter((x) => x.gender === "f"),
-  //              ^? 😔 'x' is of type 'unknown'
+  //              ^? (parameter) x: { name: string, age: number; gender: string }
   R.groupBy((x) => x.age),
-  //                 ^? 😔 Property 'age' does not exist on type 'P'
+  //                 ^? (property) age: number
```

Comment:
I was reading through the docs and the remeda section seemed to be a
copy/paste mistake from the lodash/fp section below. I copied the
example over to TS Playground to verify that TS does in fact properly
infer the types.

Reply:
ah, whoops. i think this is actually supposed to be for Ramda, can you fix?